src/lib/y2storage/disk_analyzer.rb

```diff
@@ -178,10 +178,14 @@ def data_for(*disks, data)
     #   All disk devices by default.
     # @return [Array<Dasd, Disk>]
```

Comment:
The return value could be any disk device, a software raid and a bcache, so I would document return as:

```
 # @return [Array<Dasd, Disk, Multipath, DmRaid, MdMember, Md, Bcache>] a disk device, software RAID or Bcahe
```

Or in a more general way:

```
 # @return [Array<BlkDevice>] a disk device, software RAID or Bcahe
```